pyccl/halos/halo_model.py

```diff
@@ -330,7 +330,7 @@ def I_1_2(self, cosmo, k, a, prof1, prof_2pt, prof2=None):
              value of `k`.
         """
         # Compute mass function
-        self._get_ingredients(a, cosmo, False)
+        self._get_ingredients(a, cosmo, True)
```

Comment:
was this a bug?

Reply:
Yes. You need to load the bias to compute `I_1_2`. If you don't pass `True` only the mass function is loaded. I guess in other cases it worked because it was loaded before for some other call.